packages/fxa-auth-server/lib/sentry.js

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Hoek = require('@hapi/hoek');
+const Sentry = require('@sentry/node');
+
+const getVersion = require('./version').getVersion;
+
+const UIDREGEX = /[a-fA-F0-9]{32}/gi;
```

Comment:
This will also match various other kinds of token, e.g. session token ids and oauth tokens and what-not. Those things probably won't be showing up in any of the places you're filtering though, and if they do, well it's probably the right thing to filter those as well!

Reply:
Renaming to reflect that with a comment, any other places in the request data that such a token might be?

Reply:
I think you covered them - we're already not logging the obvious things like request body and auth headers, right?

Reply:
Auth header, no. Body might be captured. 🤔

Reply:
Updated to remove request data entirely.